tests/kani/BoundedArbitrary/vec.rs

```diff
@@ -0,0 +1,56 @@
+// Copyright Kani Contributors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+// kani-flags: --default-unwind 17
```

Comment:
You can annotate individual harnesses with `#[kani::unwind(N)]`. For example, `check_reverse_is_its_own_inverse` only needs an unwinding of 5?